src/main.rs

```diff
@@ -107,8 +107,8 @@ impl Kopium {
                     if s.level == 1 && s.name.ends_with("Spec") {
                         self.print_derives();
                         println!(
-                            r#"#[kube(group = "{}", version = "{}", kind = "{}", plural = "{}")]"#,
-                            group, version, kind, plural
+                            r#"#[kube(group = "{}", version = "{}", kind = "{}", plural = "{}", status = "{}Status")]"#,
```

Comment:
this is right, but it needs to be done conditionally. it needs to check in the crd type if the status subresource is set first

Reply:
ah, good point! Pushing a new version